src/OpenTelemetry.Exporter.Jaeger/Implimentation/JaegerLog.cs

```diff
@@ -0,0 +1,100 @@
+// <copyright file="JaegerLog.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.Jaeger.Implimentation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Thrift.Protocols;
+    using Thrift.Protocols.Entities;
+
+    public class JaegerLog : TAbstractBase
+    {
+        public JaegerLog()
+        {
+        }
+
+        public JaegerLog(long timestamp, List<JaegerTag> fields)
+            : this()
+        {
+            this.Timestamp = timestamp;
+            this.Fields = fields;
+        }
+
+        public long Timestamp { get; set; }
+
+        public List<JaegerTag> Fields { get; set; }
```

Comment:
Looks like they are just enumerated. Could it be `IEnumerable<T>`?
Also, looks like it'll throw `NullReferenceException` if the parameterless ctor is called and the property setter is never called after that.
If not possible perhaps just `IList` or even `ICollection`.